### PR TITLE
[front] enh: allow large CSV/XLSX/PPTX skill attachments

### PR DESCRIPTION
**Pull request by id13**
## Description

Make it possible to upload large data/delimited files for skills attachments.

Checked everywhere and except for a grep edge case (single-lined huge files) we should be good on the "don't leak 350MB of data in the context window".

Added some tests too

## Tests

Unit tests.

## Risk

Low

## Deploy Plan

- Deploy front

### CODE REVIEW

**Comment by id13** [comment]
This change is part of the following stack:

- #27907
    - #27908 ◀

<sub>Change managed by [git-spice](https://abhinav.github.io/git-spice/).</sub>

File: front/components/skill_builder/SkillBuilderFilesSection.tsx [hunk 1]
```diff
@@ -43,7 +46,7 @@ export function SkillBuilderFilesSection({
   const fileListBottomSentinelRef = useRef<HTMLDivElement>(null);
 
   const { handleFilesUpload, isProcessingFiles } = useFileUploaderService({
-    hasSandboxTools: false,
+    hasSandboxTools: isComputerFeatureEnabled(featureFlags),
```
Comment (id13) [inline front/components/skill_builder/SkillBuilderFilesSection.tsx:49]: this

File: front/types/files.ts [hunk 2]
```diff
@@ -320,6 +322,8 @@ export function allowsSandboxRawUpload({
   switch (useCase) {
     case "conversation":
       return category === "delimited";
+    case "skill_attachment":
+      return category === "delimited" || category === "data";
```
Comment (id13) [inline front/types/files.ts:328]: and this

Comment (id13) [reply front/types/files.ts:328]: are the only changes

the rest is tests

**Comment by id13** [comment]
PMRR @flvndvd

**Review by flvndvd — COMMENTED** [review]
LGTM
